packages/openapi-to-graphql/lib/oas_3_tools.d.ts

```diff
@@ -156,15 +156,20 @@ export declare function getSecuritySchemes(oas: Oas3): {
 export declare function getSecurityRequirements(path: string, method: string, securitySchemes: {
     [key: string]: ProcessedSecurityScheme;
 }, oas: Oas3): string[];
+export declare enum CaseStyle {
+    PascalCase = 0,
+    camelCase = 1,
+    ALL_CAPS = 2
+}
 /**
  * First sanitizes given string and then also camel-cases it.
  */
-export declare function sanitize(str: string, lowercaseFirstChar?: boolean): string;
+export declare function sanitize(str: string, caseStyle: CaseStyle): string;
```

Comment:
Sanitize now takes in a case. Before we were changing the case everywhere it was not very clear.